README.md

```diff
@@ -43,14 +47,58 @@ pip3 install -r requirements.txt
 
 MacOS / Linux:
 ```
-$ export FLASK_APP=toynet-flask.py
+$ export FLASK_APP=flasksrc
+
+# restarts server after code changes
+$ export FLASK_ENV=development
+
+# creates instance/flaskr.sqlite
+$ flask init-db 
+Initialized the database.
+
 $ flask run
- * Running on http://127.0.0.1:5000/
+ * Serving Flask app "flasksrc" (lazy loading)
+ * Environment: development
+ * Debug mode: on
+ * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
+ * Restarting with stat
+ * Debugger is active!
+ * Debugger PIN: 220-725-712
  ```
 
 Windows:
 ```
-C:\path\to\app>set FLASK_APP=hello.py
+> set FLASK_APP=flasksrc
+> set FLASK_ENV=development
+> flask init-db
+> flask run
+```
+
+Windows PowerShell:
+```
+> $env:FLASK_APP = "flasksrc"
+> $env:FLASK_ENV = "development"
+> flask init-db
+> flask run
 ```
 
-![Running App](https://github.com/Project-Reclass/toynet-flask/blob/main/images/hello-reclass.png)
\ No newline at end of file
+Go to: `http://127.0.0.1:5000/`
+![Running App](../documentation/images/hello-reclass.png)
+
+Go to: `http://127.0.0.1:5000/values/5001`
+![Running App](../documentation/images/values-goodID.png)
+Open Network tab of Chrome DevTools (right click screen & click "Inspect")
+![Running App](../documentation/images/values-goodID-200.png)
+
+Go to: `http://127.0.0.1:5000/values/1`
+![Running App](../documentation/images/values-badID.png)
+Open Network tab of Chrome DevTools (right click screen & click "Inspect")
+![Running App](../documentation/images/values-badID-404.png)
+
```

Comment:
I would also add a small section about getting out of the virtual environment. Or maybe in the `Troubleshooting` section. Just in case someone get's confused by it.

Reply:
Put in the venv section!